flink-runtime-web/src/main/resources/META-INF/NOTICE

```diff
@@ -1053,13 +1053,13 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 ---
 color-name@1.1.4
 ---
-The MIT License (MIT)
-Copyright (c) 2015 Dmitry Ivanov
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the &quot;Software&quot;), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-

```

Comment:
nit: can we remove this as it seems to be removing and adding the same content. Or is there some intended change here?